docs/snippets/snippets.toml

```diff
@@ -135,7 +135,9 @@ quick_start = [ # These examples don't have exactly the same implementation.
   "py",
   "rust",
 ]
-
+"archetypes/transform3d_hierarchy" = [ # Uses a lot of trignometry which is suprisingly easy to get the same on Rust & C++, but not on Python/Numpy
```

Comment:
I remember having to deal with these kinds of shenanigans in the past -- iirc it's because numpy defaults to f64 for every computation internally. I don't remember if there's an easy way to change that.

Reply:
yeah I tried to tell it to use f32 on its arrays, but it got fiddly - definitely got closer but not to the exact result